waku/v2/protocol/pb/waku_message.proto

```diff
@@ -2,6 +2,16 @@ syntax = "proto3";
 
 package pb;
 
+message WakuMessage {
+    bytes payload = 1;
+    string contentTopic = 2;
+    uint32 version = 3;
+    sint64 timestamp = 10;
+    bytes meta = 11;
+    bool ephemeral = 31;
+    bytes rate_limit_proof = 21;
+}
+
 message RateLimitProof {
```

Comment:
And this is no longer required then? 
Otherwise let us add it to common proto repo itself and source these definitions from there as well.

If not as part of this PR, but a subsequent one.